torchci/test/mergeCommands.test.ts

```diff
@@ -106,7 +106,10 @@ describe("merge-bot", () => {
         );
         return true;
       })
-      .reply(200, {});
+      .reply(200, {})
```

Comment:
Aside: Mocking new paths is starting to require too much boilerplate in these tests. We could use a new abstraction here.